nestjs/src/auto-test/auto-test.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { Controller, Get, NotFoundException, Param, ParseIntPipe } from '@nestjs/common';
+import { AutoTestService } from './auto-test.service';
+import { ApiOkResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+import { CourseRole } from '@entities/session';
+import { RequiredRoles, Role } from 'src/auth';
+import { AutoTestTaskDto } from './dto/auto-test-task.dto';
+import { MinimizedAutoTestTaskDto } from './dto/minimized-auto-test-task.dto';
+
+@Controller('auto-test')
+@ApiTags('auto-tests')
+export class AutoTestController {
+  constructor(private readonly service: AutoTestService) {}
+
+  @Get()
+  @RequiredRoles([Role.Admin, CourseRole.Manager])
+  @ApiOperation({ operationId: 'getAllMinimizedRSSchoolAppTests' })
+  @ApiOkResponse({ type: [MinimizedAutoTestTaskDto] })
+  async getAllMinimizedRSSchoolAppTests() {
```

Comment:
lets remove `RSSchoolApp` from the name as it looks redundant

Reply:
Also I suggest another name. `minimized` looks confusing here as it's usually used for describing UI patterns.

What about the following names:
- `getAllAutoTests()`
- `getBasicAutoTests()`